### PR TITLE
Update to Swagger UI 3.9.1

### PR DESCRIPTION
**Pull request by BlackBrane**
I bumped us a few versions of Servant Swagger.

Incidentally this required a change to the GHC options in the cabal file to allow more memory for building, otherwise it would stack overflow.

I also updated the stack configuration to use LTS-10.3 / GHC 8.2.2. 

I was motivated to do this change when I tried using the UI on the swagger website and noticed that it was substantially less laggy. I haven't noticed many other changes yet, but that one was significant enough to be worth the upgrade.

Been using it actively for a couple days, and will continue to.

### CODE REVIEW

File: servant-swagger-ui.cabal [hunk 1]
```diff
@@ -69,12 +68,18 @@ extra-source-files:
...
+    swagger-ui-dist-3.9.1/favicon-16x16.png
+    swagger-ui-dist-3.9.1/favicon-32x32.png
+    swagger-ui-dist-3.9.1/index.html
+    swagger-ui-dist-3.9.1/oauth2-redirect.html
+    swagger-ui-dist-3.9.1/swagger-ui-bundle.js
+    swagger-ui-dist-3.9.1/swagger-ui-bundle.js.map
+    swagger-ui-dist-3.9.1/swagger-ui-standalone-preset.js
+    swagger-ui-dist-3.9.1/swagger-ui-standalone-preset.js.map
+    swagger-ui-dist-3.9.1/swagger-ui.css
+    swagger-ui-dist-3.9.1/swagger-ui.css.map
```
Comment (phadej) [inline servant-swagger-ui.cabal]: Please remove map files, without them you won't need to tweak GHC memory (which I won't accept anyway, that should be configured externally, e.g. In `stack.yaml`)

Comment (BlackBrane) [reply servant-swagger-ui.cabal]: Makes sense. I did this, and also made another small mention in the README instructions about it.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Check [`example/Main.hs`](https://github.com/phadej/servant-swagger-ui/blob/mast
 
 ### Updating of bundled swagger-ui version:
 
-- Extract `dist` directory of `swagger-ui` into `swagger-dist-&lt;version&gt;`
+- Extract `dist` directory of `swagger-ui` into `swagger-dist-&lt;version&gt;`. Remove the `*.map` files.
```
Comment (phadej) [inline README.md]: :+1:

**Comment by phadej** [comment]
I'm in a process of preparing `servant-0.13` release. I'll merge and release after I'm done with it.

**Comment by BlackBrane** [comment]
Great. Thanks for your efforts!

**Pull request merged**